src/EditorFeatures/CSharpTest/CrashHardTest.cs

```diff
@@ -0,0 +1,19 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.UnitTests
+{
+    public class CrashHardTest
+    {
+        [Fact, Trait(Traits.Feature, "Crash")]
+        public void Crash()
+        {
+            ExceptionUtilities.FailFast(new Exception("Goodbye!"));
```

Comment:
Shouldn't this really be saying `"Hello"` instead?  After all this is going to pop up in a developer's email eventually, may as well have a greeting to go with it vs. a dismissal.  :trollface: